Example/SmileID/RootView.swift

```diff
@@ -4,9 +4,12 @@ import SmileID
 struct RootView: View {
     // This is set by the SettingsView
     @AppStorage("smileConfig") private var configJson = (
-        UserDefaults.standard.string(forKey: "smileConfig") ?? ""
+            UserDefaults.standard.string(forKey: "smileConfig") ?? ""
     )
     private let jsonDecoder = JSONDecoder()
+    @State private var showSuccess = false
+    @State private var partnerId = ""
```

Comment:
Would be much better if this was a nullable String, and not assigned to an empty string

Reply:
Fixed